src/module.ts

```diff
@@ -75,8 +75,8 @@ export enum FeatureFlags {
   MultiValue = 512 /* _BinaryenFeatureMultivalue */,
   GC = 1024 /* _BinaryenFeatureGC */,
   Memory64 = 2048 /* _BinaryenFeatureMemory64 */,
-  TypedFunctionReferences  = 4096 /* _BinaryenTypedFunctionReferences */,
-  All = 8191 /* _BinaryenFeatureAll */
+  TypedFunctionReferences = 4096 /* _BinaryenFeatureTypedFunctionReferences */,
+  All = 16383 /* _BinaryenFeatureAll */
```

Comment:
Seems there's a new that needs to be added

Reply:
That's experimental non-nullable flags for GC. It is not known if it will be included in the final wasm GC proposal. I suggest that you take your time adding this

Reply:
I see, thanks